hgtv/views/video.py

```diff
@@ -190,13 +189,31 @@ def make_presentz_json(video, json_value):
         unique_value = get_slideshare_unique_value(video.slides_url)
         d['chapters'][0]['slides'] = [{'time': str(key), "public_url": urllib.quote(video.slides_url), "url": 'https://slideshare.net/' + unique_value + "#" + str(val)} for key, val in json_value.items()]
     elif video.slides_source == u'speakerdeck':
-        #json to supply for presentz syncing
+        # json to supply for presentz syncing
```

Comment:
I think we can remove all the Presentz-related code. It's not been used in many years.